### ISSUE TITLE
issue creating admin user - 1st install

### ISSUE THREAD

**Issue by h0wlermonkey** [sjarvela/kloudspeaker#127]
hello

I've completed the database & user setup, setting the admin login and I receive the following error message:

Could not install

Error executing queries (CREATE TABLE `user` ( `id` int(11) NOT...): All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead

**Comment by Tjr25** [comment]
I had the same issue just now. I'm using mysql. I fixed it by changing the file \kloudspeaker\backend\db\mysql\sql\install\create_tables.sql

Use a text editor like Notepad++ rather than just Notepad.

Go to line 48.

Change it to
`user_id` int(11) NOT NULL DEFAULT 0,

Or just replace your file with the one I have attached.
[create_tables.zip](https://github.com/sjarvela/kloudspeaker/files/249660/create_tables.zip)

**Comment by h0wlermonkey** [comment]
That did the trick!! thanks!

---- On Wed, 04 May 2016 22:04:09 +0000 Tjr25 &lt;notifications@github.com&gt; wrote ---- 

I had the same issue just now. I'm using mysql. I fixed it by changing the file \kloudspeaker\backend\db\mysql\sql\install\create_tables.sql
 Use a text editor like Notepad++ rather than just Notepad.
 Go to line 48.
 Change it to
 user_id int(11) NOT NULL DEFAULT 0,
 Or just replace your file with the one I have attached.
 create_tables.zip
